app/javascript/src/components/PlayerEnd.tsx

```diff
@@ -17,11 +18,20 @@ export const PlayerEnd: React.SFC<Props> = ({ title, body, image, onReplay, onGo
     }
   }
 
+  const sourceButton = () => {
```

Comment:
I like to prefix the name of function that returns JSX with `render` to make it clear that it's returning some part of a rendered component.